kensu/pyspark/spark_connector.py

```diff
@@ -488,7 +488,7 @@ def init_kensu_spark(
         try:
             config.read(conf_path)
         except:
-            logging.warning(f"Cannot load config from file `%s`" % (conf_path))
+            logging.error(f"Cannot load config from file `%s`" % (conf_path))
```

Comment:
```suggestion
            logging.error(f"Cannot load Kensu config from file `%s`" % (conf_path))
```

Reply:
@leszekgruchala this is the only exception i agree with, conf file changes rarely, and is not impacted by us directly, so useful to catch/notice conf issues